util/cosmos/iscn/query.js

```diff
@@ -145,9 +154,33 @@ export async function getISCNTransferInfo(txHash, opt) {
       owner,
     });
   });
+  if (isLIKETx) {
+    return { isLIKETx };
+  }
   const [message] = messages;
   if (!message) {
-    throw new Error('Not an ISCN transaction');
+    const txRaw = TxRaw.decode(Buffer.from(txData.tx, 'base64'));
+    const txBody = TxBody.decode(txRaw.bodyBytes);
+    const bodyMessages = txBody.messages;
+    if (bodyMessages.length > 1) {
+      bodyMessages.forEach((m) => {
+        const { typeUrl } = m;
+        if (typeUrl === '/cosmos.bank.v1beta1.MsgSend' || typeUrl === '/cosmos.bank.v1beta1.MsgMultiSend') {
+          isLIKETx = true;
+          return { isLIKETx };
+        }
+        return new Error('Not an known transaction type');
+      });
+    } else if (bodyMessages.length === 1) {
+      const { typeUrl } = bodyMessages[0];
+      if (typeUrl === '/cosmos.bank.v1beta1.MsgSend' || typeUrl === '/cosmos.bank.v1beta1.MsgMultiSend') {
+        isLIKETx = true;
+        return { isLIKETx };
+      }
+      return new Error('Not an known transaction type');
+    } else {
+      return new Error('Not an known transaction type');
+    }
```

Comment:
Why do we need this part if we already check typeUrl in line 131?

Reply:
because line 131 actually did not have any effect....due to parsed.tx.body.messages = []
Will revise the code!